### PR TITLE
add Readme for openebs-monitoring

### LINKED ISSUE #390: Add Documentation for prometheus
## Documentation
* Add Readme and documentation related to prometheus.

### PR DESCRIPTION
**Pull request by utkarshmani1997**
**What this PR does / why we need it**:
* This PR refers to issue #390
* This PR adds Readme for the openebs-monitoring. It has all the
  instructions about how to launch prometheus.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #390 

**Special notes for your reviewer**:
* None

### CODE REVIEW

File: k8s/openebs-monitoring/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+## Overview
+* [Prometheus](https://github.com/prometheus/prometheus) is a monitoring tool  which is used to monitor OpenEBS volumes. There is already a lot of exporters like [node-exporter](https://github.com/prometheus/node_exporter) are available which can be used to gather the various metrics.
```
Comment (mahebbar) [inline k8s/openebs-monitoring/README.md]: There are already a log of exporters like...
remove 'are available'
remove 'the' from the various metrics.

File: k8s/openebs-monitoring/README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+## Overview
+* [Prometheus](https://github.com/prometheus/prometheus) is a monitoring tool  which is used to monitor OpenEBS volumes. There is already a lot of exporters like [node-exporter](https://github.com/prometheus/node_exporter) are available which can be used to gather the various metrics. 
+* This subdir has two files
```
Comment (mahebbar) [inline k8s/openebs-monitoring/README.md]: subdir = sub directory

File: k8s/openebs-monitoring/README.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    - prometheus.yaml
+    - prometheus-operator.yaml
+* prometheus.yaml :  This is the configuration file for prometheus, which can be used by prometheus to get the targets (Where to collect metrics?) and it displays the collected metrics in Prometheus expression browser.
```
Comment (mahebbar) [inline k8s/openebs-monitoring/README.md]: to get targets = to receive targets?
(Where...?)

File: k8s/openebs-monitoring/README.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+* prometheus-operator.yaml : This is the operator file for prometheus which is used to launch prometheus and it's components on [Kubernetes](https://github.com/kubernetes/kubernetes) cluster.
+
+## Prerequisits
+* Please go through the steps mentioned in [k8s/vagrant](https://github.com/openebs/openebs/tree/master/k8s/vagrant) subdir .
```
Comment (mahebbar) [inline k8s/openebs-monitoring/README.md]: Remove please
steps mentioned = procedure given

File: k8s/openebs-monitoring/README.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+## Setup Prometheus
+
+Fetch the latest prometheus-operator.yaml
+```
+wget https://raw.githubusercontent.com/openebs/openebs/master/k8s/openebs-monitoring/prometheus.yaml
+wget https://raw.githubusercontent.com/openebs/openebs/master/k8s/openebs-monitoring/prometheus-operator.yaml
+```
+Create configmap and launch prometheus-operator on your kubernetes cluster
```
Comment (mahebbar) [inline k8s/openebs-monitoring/README.md]: Kubernetes

File: k8s/openebs-monitoring/README.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+```
+kubectl create -f prometheus.yaml
+kubectl create -f prometheus-operator.yaml
+```
+## Verify
+After Successful run of the above commands  you will see output like below :
```
Comment (mahebbar) [inline k8s/openebs-monitoring/README.md]: ;After successfully running the above commands, the output displayed is similar to the following:

File: k8s/openebs-monitoring/README.md [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+ubuntu@kubemaster-01:~$ kubectl create -f prometheus-operator.yaml
+serviceaccount "prometheus" created
+clusterrole "prometheus" created
+clusterrolebinding "prometheus" created
+deployment "prometheus-deployment" created
+service "prometheus-service" created
+daemonset "node-exporter" created
+```
+## Launch Prometheus Expression Browser
+* Do `kubectl cluster-info` to get the NodeIP
```
Comment (mahebbar) [inline k8s/openebs-monitoring/README.md]: Do = Run

File: k8s/openebs-monitoring/README.md [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+* Open NodeIP:32514/targets in your Browser  and it will launch prometheus expression browser.
```
Comment (mahebbar) [inline k8s/openebs-monitoring/README.md]: browser

**Comment by utkarshmani1997** [comment]
thanks @mahebbar , i will update it.

**Pull request merged**